app/assets/stylesheets/question_index.scss

```diff
@@ -0,0 +1,25 @@
+.question-body {
+    font-size: larger;
+    font-family: -webkit-body;
+  }
+
+  .button-round {
+    margin-right: 347px;
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.